tutorials/sample_vqe_program/qiskit_runtime_vqe_program.ipynb

```diff
@@ -0,0 +1,1394 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "dangerous-croatia",
+   "metadata": {},
+   "source": [
+    "# Creating Custom Programs for the Qiskit Runtime\n",
+    "\n",
+    "<p>\n",
+    "<font size=\"4\" color=\"#0f62fe\">Paul Nation</font>\n",
+    "</p>\n",
+    "<p>\n",
+    "<font size=\"3\" color=\"#0f62fe\">IBM Quantum Partners Technical Enablement Team</font>\n",
+    "</p>\n",
+    "\n",
+    "Here we will demonstrate how to create, upload, and use a custom Program for the Qiskit Runtime.  As the utility of the Runtime execution engine lies in its ability to execute many quantum circuits with low latencies, this tutorial will show how to create your own Variational Quantum Eigensolver (VQE) program from scratch."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "sticky-annual",
+   "metadata": {},
+   "source": [
+    "## Prerequisites\n",
+    "\n",
+    "- You must have Qiskit 0.30+ installed.\n",
+    "- You must have an IBM Quantum Experience account with the ability to upload a Runtime program.  **Currently there is no way to know if you have Runtime upload ability outside of an email from IBM**."
```

Comment:
I am not sure about this statement.  Are the Africa challenge participants considered premium because they were added to another provider?  Or members of the Qiskit community hub?